src/main/resources/application.yaml

```diff
@@ -34,3 +34,12 @@ idam:
     id: 'bsp'
     secret: '123456'
     redirect_uri: 'http://localhost/receiver'
+
+logging:
+  level:
+    uk.gov.hmcts.reform.bulkscan: WARN
+    org.springframework: WARN
+    com.microsoft.azure.servicebus: DEBUG
+    org.apache.qpid: DEBUG
+    org.apache.catalina: WARN
+    springfox.documentation: WARN
```

Comment:
perhaps this could be configurable using env vars (I assume we don't want those on prod?)

Reply:
No, I was planning to remove it altogether, but you are right it could be made configurable.